src/wallet/wallet.cpp

```diff
@@ -48,6 +48,8 @@ bool fSendFreeTransactions = DEFAULT_SEND_FREE_TRANSACTIONS;
  */
 CFeeRate CWallet::minTxFee = CFeeRate(DEFAULT_TRANSACTION_MINFEE);
 
+const uint256 CMerkleTx::abandonHash(uint256S("0000000000000000000000000000000000000000000000000000000000000001"));
```

Comment:
> I debated doing that and maybe it would have been better, but I liked the way updating the hashBlock automatically made the transaction no longer abandoned with this approach.

fair enough.